x/crosschain/keeper/v2_zevm_inbound.go

```diff
@@ -49,7 +50,7 @@ func (k Keeper) ProcessZEVMInboundV2(
 			contractAddress = withdrawalAndCallEvent.Raw.Address
 		}
 
-		k.Logger(ctx).Error(fmt.Sprintf("processing inbound. zrc20: %s", zrc20.Hex()))
+		k.Logger(ctx).Info(fmt.Sprintf("processing inbound. zrc20: %s", zrc20.Hex()))
```

Comment:
```suggestion
```

We can actually remove this log. It was used in the past for debugging when implementing v2 workflow